.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+  push:
+    branches:
+      # TODO: cover hotfix, release, master, etc?
+      - develop
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - "*"
+  schedule:
+    # Daily at 09:37
+    - cron: '37 9 * * *'
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - name: CPython 3.9
+            version: 3.9
+            action: 3.9
+        arch:
+          - name: x64
+            action: x64
+            matrix: x64
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Python ${{ matrix.python.name }} ${{ matrix.arch.name }} for Windows
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python.action }}
+          architecture: ${{ matrix.arch.action }}
+
+      - name: boots
+        run: |
+          ./boots.py ensure
+
+      - name: Package executable
+        run: |
+          ./venv/Scripts/activate
```

Comment:
Activating varies per shell both in the sense that in bash you need to `source venv/Scripts/activate` and that `cmd.exe` needs `venv/Scripts/activate.bat` while PowerShell needs `venv/Scripts/activate.ps1`.  Regardless, we might want to just force bash in all cases.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun